### PR TITLE
Backport of missing commits to 12.0

### PR DESCRIPTION
**Pull request by etobella**

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Hi @etobella why there are two menus to iot.device? https://github.com/OCA/iot/pull/55/files#diff-f81eb1631a021b2971518b1c8ae043a83bdcf7cc8567b96851164aacc1c8bb87R243
can we remove one?

**Comment by etobella** [comment]
> Hi @etobella why there are two menus to iot.device? https://github.com/OCA/iot/pull/55/files#diff-f81eb1631a021b2971518b1c8ae043a83bdcf7cc8567b96851164aacc1c8bb87R243 can we remove one?

Good catch!

I think it will be green now. Any reviews are welcome

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

File: iot_input/views/iot_device_input_views.xml [hunk 1]
```diff
@@ -66,7 +66,6 @@
         <field name="name">IoT Inputs</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">iot.device.input</field>
-        <field name="view_type">form</field>
```
Comment (MiquelRForgeFlow) [inline iot_input/views/iot_device_input_views.xml]: In v12 you still have this. Is in v13 that got deleted.

Comment (etobella) [reply iot_input/views/iot_device_input_views.xml]: Done!

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**